### PR TITLE
Ip whitelist entries: removing all entries whitelist by terraform user

### PR DESCRIPTION
**Pull request by PacoDw**
The following changes are made:

- Avoid removing all entries whitelist by terraform user

### CODE REVIEW

File: mongodbatlas/resource_mongodbatlas_project_ip_whitelist.go [hunk 1]
```diff
@@ -103,13 +103,13 @@ func resourceMongoDBAtlasProjectIPWhitelistCreate(d *schema.ResourceData, meta i
...
 	req := expandProjectIPWhitelist(d)
-	resp, _, err := conn.ProjectIPWhitelist.Create(context.Background(), projectID, req)
+	_, _, err := conn.ProjectIPWhitelist.Create(context.Background(), projectID, req)
 	if err != nil {
 		return fmt.Errorf("error creating project IP whitelist: %s", err)
 	}
 
 	var withelist []string
-	whiteListMap(resp, func(entry string) {
+	whiteListMap(req, func(entry string) {
```
Comment (marinsalinas) [inline mongodbatlas/resource_mongodbatlas_project_ip_whitelist.go:112]: why did you change req instead de response?

Comment (PacoDw) [reply mongodbatlas/resource_mongodbatlas_project_ip_whitelist.go:112]: because response get all whitelist entries 👍

**Review by themantissa — APPROVED** [review]

**Review by themantissa — APPROVED** [review]
Make sure Marin takes a look as well just for a double check please.

**Review by marinsalinas — APPROVED** [review]
